### PR TITLE
Adding Sentinel automation rule template

### PR DESCRIPTION
**Pull request by javiersoriano**
# PR Checklist

Check these items before submitting a PR... 

[Contribution Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md)

[Best Practice Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md)

- [X ] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

## Changelog

Adding new template for Sentinel Automation Rules

### CODE REVIEW

File: quickstarts/microsoft.securityinsights/sentinel-automation-rule/README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Creates an Automation Rule in an existing Microsoft Sentinel workspace #
```
Comment (bmoore-msft) [inline quickstarts/microsoft.securityinsights/sentinel-automation-rule/README.md:1]: use title casing

File: quickstarts/microsoft.securityinsights/sentinel-automation-rule/README.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+[![Visualize](https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/1-CONTRIBUTION-GUIDE/images/visualizebutton.svg?sanitize=true)](http://armviz.io/#/?load=https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2Fquickstarts%2Fmicrosoft.securityinsights%2Fsentinel-automation-rule%2Fazuredeploy.json)
+
+This sample template demonstrates how to create an Automation Rule in your Microsoft Sentinel workspace. This sample automation rule triggres on incident creation and looks for specific analytic rule ID, severity, tactics and title. If the incident matches these conditions, it then executes a playbook and modifies incident status and adds a tag. For more information about automation rules, visit [Automation in Azure Sentinel](https://docs.microsoft.com/en-us/azure/sentinel/automation-in-azure-sentinel)
```
Comment (bmoore-msft) [inline quickstarts/microsoft.securityinsights/sentinel-automation-rule/README.md:16]: typo ```triggers```

File: quickstarts/microsoft.securityinsights/sentinel-automation-rule/prereqs/prereq.azuredeploy.json [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+                }
+            }
+        },
+        {
+            "name": "[concat(parameters('workspaceName'),'/Microsoft.SecurityInsights/default')]",
+            "type": "Microsoft.OperationalInsights/workspaces/providers/onboardingStates",
+            "apiVersion": "2021-03-01-preview",
+            "properties": {
+            },
+            "dependsOn": [
```
Comment (bmoore-msft) [inline quickstarts/microsoft.securityinsights/sentinel-automation-rule/prereqs/prereq.azuredeploy.json:47]: move this above properties

**Review by bmoore-msft — CHANGES_REQUESTED** [review]
Also check the pipeline results for BP changes that need to be made

**Comment by javiersoriano** [comment]
@bmoore-msft let me know if this is good now. Thanks!

**Pull request merged**